ioos_qartod/qc_tests/qc.py

```diff
@@ -177,6 +181,16 @@ def spike_check(arr, low_thresh, high_thresh, prev_qc=None):
     flag_arr = ((val < low_thresh) +
                 ((val >= low_thresh) & (val < high_thresh)) * QCFlags.SUSPECT +
                 (val >= high_thresh) * QCFlags.BAD_DATA)
+
+    # Change flag to unknown for data around large time gaps that failed the spike test
+    dt = np.mean(np.diff(times))  # mean value time between data samples
```

Comment:
honestly, I don't even know if 1σ is the right answer here either. 

![obs-data](https://cloud.githubusercontent.com/assets/1155163/20282978/190857ba-aa85-11e6-9823-e97e391b946d.png)

![timediffs](https://cloud.githubusercontent.com/assets/1155163/20282980/1bfab33c-aa85-11e6-9224-38a391b08728.png)